source/layouts/layout.slim

```diff
@@ -6,6 +6,9 @@ html
     meta charset="utf-8" /
     meta content="IE=edge;chrome=1" http-equiv="X-UA-Compatible" /
     meta content="width=device-width, initial-scale=1" name="viewport" /
+    meta content="SABEResPODER te ayuda prosperar en los EEUU, con recursos y contestas personalizadas a tus preguntas sobre inmigración, educación, finanzas, salud y otros asuntos importantes. Tu causa es nuestra causa!" name="description" /
+    meta content="Inmigracion, ciudania, preguntas, articulos, derechos, finanzas, salud, educacion, comunidad, consulado, visa, pasaporte, estados unidos, informacion, ofertas, viviendo en los estados unidos, trabajar, trabajo, impuestos, taxes, embajada, el salvador, mexico, guatemala, informacion gratuita, medicamentos, farmacias, farmasep, descuentos, encuestas" name="keywords" /
```

Comment:
Thanks David. It is interesting because in #12 @stan2020 listed keywords exactly in this way...